SodaInventory/Controllers/InventoryController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SodaInventory.Model;
+
+namespace SodaInventory.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class InventoryController : ControllerBase
+    {
+        private readonly DatabaseContext _context;
+
+        public InventoryController(DatabaseContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/GetInventory
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<object>>> GetInventory()
```

Comment:
Didn't we decide that the dashboard would only show for the current store and that we would make an aggregate dashboard later?

Reply:
Yes, I added a parameter for the storeId I missed this part.